### PR TITLE
20210511 zrq hdfs transfers and data shares

### PR DESCRIPTION
**Pull request by Zarquan**
Transferring repartitioned `GEDR3_4096` and `GEDR3_4096` parquet files from HDFS.
Reconfiguring the directory structure for the Gaia data.
Reconfiguring the Manila shares, adding new shares for the repartitioned data and adding prefixes to the names.

These changes are not backwards compatible - the Manila share names have changed so old deployments won't be able to mount them.

### CODE REVIEW

File: deployments/hadoop-yarn/ansible/config/zrq-dev-small.yml [hunk 1]
```diff
@@ -0,0 +1,172 @@
+#
+# <meta:header>
+#   <meta:licence>
+#     Copyright (c) 2020, ROE (http://www.roe.ac.uk/)
```
Comment (stvoutsin) [inline deployments/hadoop-yarn/ansible/config/zrq-dev-small.yml:4]: 2020 -> 2021

File: notes/zrq/20210510-03-hdfs-transfer.txt [hunk 2]
```diff
@@ -0,0 +1,999 @@
+#
+# <meta:header>
+#   <meta:licence>
+#     Copyright (c) 2020, ROE (http://www.roe.ac.uk/)
```
Comment (stvoutsin) [inline notes/zrq/20210510-03-hdfs-transfer.txt:4]: 2020 -> 2021

**Review by stvoutsin — APPROVED** [review]

File: deployments/hadoop-yarn/ansible/config/zrq-dev-small.yml (hunk 1 above)
Comment (Zarquan) [reply deployments/hadoop-yarn/ansible/config/zrq-dev-small.yml:4]: Well caught

File: notes/zrq/20210510-03-hdfs-transfer.txt (hunk 2 above)
Comment (Zarquan) [reply notes/zrq/20210510-03-hdfs-transfer.txt:4]: Yep, my bad.

**Pull request merged**